__tests__/mindfulness.test.tsx

```diff
@@ -6,11 +6,34 @@ jest.mock('next/navigation', () => jest.requireActual('next-router-mock'))
 jest.mock('../lib/zoomapi', () => jest.requireActual('../lib/fakezoomapi'));
 
 describe('Page', () => {
-  it('renders a heading', () => {
+  it('renders a heading and two videos', () => {
     render(<Page />)
 
     const heading = screen.getByRole('heading', { level: 1 })
 
     expect(heading).toBeInTheDocument()
+
+    const video1 = screen.getByTestId('youtube1-iframe');
```

Comment:
Rather than have a test id for each video, I think it's cleaner to have a specific role for the videos, and then use getAllByRole and assert that it has length 2.